admin_scripts/systemd_policy_cleanup.json

```diff
@@ -0,0 +1,12 @@
+ {
+  "name": "System - Slet brugermappen ved opstart og nedlukning",
+  "description": "Installerer en system-tjeneste, der sletter alt indhold i publikum-brugerens hjemmemappe ved opstart og nedlukning af systemet.\n\nUdarbejdet af Alexander Faithfull <af@magenta.dk>.",
+  "parameters": [
+    {
+      "name": "Aktivér?",
+      "type": "string",
+      "required": false,
+      "description": "system-tjenesten vil blive fjernet, hvis denne parameter er tomt eller ordet 'falsk'"
```

Comment:
correcting "dette" to "denne" (since "parameter" is fælleskøn), "tomt" should also be "tom" (sorry, but I'm a bit OCD about Danish grammar :slightly_smiling_face: )

Reply:
¿Vel jeg kunne ~verstehen~forstår den !